app-text/htmltidy/tidy-5.6.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="The granddaddy of HTML tools, with support for modern standards"
+DESCRIPTION="Tidy tidies HTML and XML. It can tidy your documents by itself, \
+and developers can easily integrate its features into even more powerful tools."
+HOMEPAGE="http://www.html-tidy.org/"
+COPYRIGHT="1998-2015 World Wide Web Consortium"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/htacg/tidy-html5/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="08a63bba3d9e7618d1570b4ecd6a7daa83c8e18a41c82455b6308bc11fe34958"
+SOURCE_DIR="tidy-html5-$portVersion"
+PATCHES="tidy-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix
+fi
+
+libVersion=$portVersion
+libVersionCompat="$libVersion compat >= 5"
+
+PROVIDES="
+	tidy$secondaryArchSuffix = $portVersion
+	cmd:tidy$commandSuffix = $portVersion
+	lib:libtidy$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	devel:libtidy$secondaryArchSuffix = $libVersionCompat
+	devel:libtidys$secondaryArchSuffix
```

Comment:
I think it should be:
```diff
-	devel:libtidys$secondaryArchSuffix
+	devel:libtidys$secondaryArchSuffix = $libVersion
```

Reply:
it's a *.a lib, no version there?

Reply:
OK then. If there is no libtidy**s**.so (or one with no .so.x.y.z) then it's ok to have no version here.

Reply:
correct :)

Reply:
Hmm, if there is no libtidy**s**.so then you are correct: lib:libtidy**s** should _not_ be in PROVIDES.